movie_recommender/movielens_movie_recommendations.ipynb

```diff
@@ -1,2632 +1,5344 @@
 {
```

Comment:
<p>please rename 'service' to 'index', and 'turn it off' to 'delete'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>